### ISSUE TITLE
Allow user admins to trigger case re-uploads with different panels to avoid re-runs

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#3668]
Discussing with production last week during the retreat, came out that it's common practice to order a complete case re-run whenever users are interested in other panels or when a panel gets a few genes added. Of course an entire re-run is not necessary and it would help immensely to reduce the workload of our computer nodes if they could just re-upload the case with different gene panels directly from Scout. This way the node resources could be dedicated to just run new cases.

The case re-upload could be achieved by
- Granting a special permission (admins?) to a few key users
- Panel-filtering the research VCFs (replacing the clinical panels with these new filtered VCFs?)
- ?

**Comment by dnil** [comment]
Would be nice, but it would still need to connect with MIP or another small service for refiltering for clinical variants. The users can already trigger research upload on which they can filter with whatever panels they like. The trade off being a lot more variants loaded. And one customer does not trust OMIM inheritance patterns but want to see their own custom ones, which they don't atm if they don't have the panel selected as "default". Perhaps we should start with fixing the latter? Like allow default selection for the panels that are optioned as always searchable?

**Comment by northwestwitch** [comment]
> Would be nice, but it would still need to connect with MIP or another small service for refiltering for clinical variants.

Yes. I've actually already written code doing exactly the same thing for the beacon package, when you filter the VCF by one or more gene panels

- First you [create a BedTool object](https://github.com/Clinical-Genomics/cgbeacon2/blob/fcc82dc9098d77389e60bd5bd13c4e2e9db91fe1/cgbeacon2/utils/parse.py#L133) with the positions of the genes of the panel.
- Then you [extract the intersections](https://github.com/Clinical-Genomics/cgbeacon2/blob/fcc82dc9098d77389e60bd5bd13c4e2e9db91fe1/cgbeacon2/utils/parse.py#L197) between the research VCF and the BedTool
- And finally you [create a new VCF files based on the intersections](https://github.com/Clinical-Genomics/cgbeacon2/blob/fcc82dc9098d77389e60bd5bd13c4e2e9db91fe1/cgbeacon2/utils/parse.py#L169)
